march_hardware_interface/include/march_hardware_interface/march_hardware_interface.h

```diff
@@ -68,8 +68,16 @@ class MarchHardwareInterface : public hardware_interface::RobotHW
    */
   int getEthercatCycleTime() const;
 
+  /**
+   * Wait for received PDO.
+   */
   void waitForPdo();
 
+  /**
+   * Construct the soft limit error based on the margin defined in the controller yaml and the soft limits and hard
+   * limits in the URDF.
+   */
+
```

Comment:
oopsie :see_no_evil: you forgot to copy the docs :joy_cat:

Reply:
I'm very dumb